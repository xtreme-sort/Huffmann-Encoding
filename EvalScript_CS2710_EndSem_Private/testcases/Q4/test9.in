GenerateLengthLimitedHuffman
vkllmhtkydibtvhaxdpvmwgrgleuhzxchytbenuymdrxxjosfhtmywqiilkraunoaviehvidncvefmefujlzljsgxiwbhmopvpygzcixqfszztikjkwqkeosbfsolllahjcwauhjxgbvwonepxkuowxkmakfeutmrnnlscxnneaegmxgxubrplwgonuruwmbbjrejvlqbxftrkbiusnvrisdgfibemkfksyekgftvwwdzdvjjqlucnncdfqqnksixmbkfpthvxxfimccdzedawfsxyzffakdgxnqzgqntnisettnzklxgkfghrdczcplxbxpnxxlsxtbvddymkykecxsqtosexqqwvwaiihtnlwjpikfzeozdzjypsouaypzsopiodtmsdtmywvsvfsyyewgtgkpxfroqplyryetxgsrvswewfybugqnkzcopafdpumplqbqvdpnfhvqulxgjekwxdadkqnlqeydurennqhqvuexajirfhvyxvglabcmfrlkavdmpbdxtflwwtlmqyfzqhkyqoxhbimumtylykakkwmbnqfndtcetsehnfvpghdvvwjoiybxtcsgtpuymrashugvslopytujltwksofzilmoscrkhkwgoovuedqlrvrjskviudxsoltqmbvgfyaikqptrhodyhicqdfhxphefjxrcxlnusmznrinwdpzivjjskixderyedzlanjdtpukppwwsuibbftjfcesunshzmgujfcdwxtfyqvdhvoiwmuzojvwcfaihe
10
Quit
