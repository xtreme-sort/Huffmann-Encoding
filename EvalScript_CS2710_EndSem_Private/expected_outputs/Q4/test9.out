d 0000
f 0001
k 0010
s 0011
v 0100
x 0101
a 01100
b 01101
c 01110
e 01111
g 10000
h 10001
i 10010
j 10011
l 10100
m 10101
n 10110
o 10111
p 11000
q 11001
r 11010
t 11011
u 11100
w 11101
y 11110
z 11111
cost: 3687
